AElf.Contracts.Consensus.Tests/ContractTestHelper.cs

```diff
@@ -0,0 +1,184 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using AElf.Common;
+using AElf.Contracts.Genesis;
+using AElf.Cryptography;
+using AElf.Cryptography.ECDSA;
+using AElf.Kernel;
+using AElf.Kernel.Account.Application;
+using AElf.Kernel.Blockchain.Application;
+using AElf.Kernel.Consensus.Application;
+using AElf.Kernel.KernelAccount;
+using AElf.Kernel.Miner.Application;
+using AElf.Kernel.Node.Application;
+using AElf.Kernel.Services;
+using AElf.Kernel.SmartContractExecution.Application;
+using AElf.Kernel.TransactionPool.Infrastructure;
+using AElf.OS;
+using AElf.OS.Account;
+using AElf.OS.Node.Application;
+using AElf.Types.CSharp;
+using Google.Protobuf;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Moq;
+using Volo.Abp;
+using Volo.Abp.DependencyInjection;
+
+namespace AElf.Contracts.Consensus.Tests
+{
+    public class ContractTestHelper : ITransientDependency
```

Comment:
ContractTestHelper => ContractTester 

Helper is usually for static stateless method